spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -52,134 +52,45 @@ http://start.spring.io/[Spring Initializr] and creating a new project. Doing so
 automatically generates a new project structure so that you can start coding right away.
 We recommend experimenting with the Spring Initializr to become familiar with it.
 
-Before we begin, open a terminal to check that you have valid versions of Java and Maven
-installed, as shown in the following two listings:
+[[getting-started-creating-project]]
+=== Creating the Spring Task Project using Spring Initializr
+Now let's create and test an application that prints `Hello, World!` to the console.
```

Comment:
"Now we can create...."

("let's" is both a contraction and more informal than we generally do.)